Tools/Firmware/CMakeLists.txt

```diff
@@ -28,6 +28,9 @@ include_directories(
     ${ROOT_DIR}/Drivers/ccontrol/Inc
     ${ROOT_DIR}/Drivers/sensor_fusion/Inc
     ${ROOT_DIR}/Drivers/IWDG_Driver/Inc
+
+    ${ROOT_DIR}/Drivers/MotorChannel/Inc
```

Comment:
is this needed for TM?

Reply:
I don't think so. I copied and pasted the file contents from the current main branch.

Reply:
Am I good to resolve this conversation?

Reply:
PWM motor control code already in main